lib/board.rb

```diff
@@ -0,0 +1,59 @@
+class Board
+  attr_accessor :cells
+  attr_reader :corners
+  @cells = []
```

Comment:
@Juanriv45 we are setting @cells to be [] in the initialize method. This isn't going to be needed.